x-pack/legacy/plugins/lens/public/indexpattern_plugin/field_item.tsx

```diff
@@ -194,31 +186,44 @@ export function FieldItem({
   };
   const seriesColors = new Map([[colors, expectedColor]]);
 
+  const totalValuesCount =
+    state.topValues && state.topValues.buckets.reduce((prev, bucket) => bucket.doc_count + prev, 0);
+  const otherCount = state.doc_count && totalValuesCount ? state.doc_count - totalValuesCount : 0;
+
+  const euiButtonColor =
+    field.type === 'string' ? 'accent' : field.type === 'number' ? 'secondary' : 'primary';
+  const euiTextColor =
+    field.type === 'string' ? 'accent' : field.type === 'number' ? 'secondary' : 'default';
+
   return (
-    <DragDrop
-      value={{ field, indexPatternId: indexPattern.id } as DraggedField}
-      data-test-subj="lnsFieldListPanelField"
-      draggable
-      className={`lnsFieldListPanel__field lnsFieldListPanel__field-btn-${
-        field.type
-      } lnsFieldListPanel__field--${exists ? 'exists' : 'missing'}`}
-    >
-      <EuiPopover
-        id="lnsFieldListPanel__field"
-        // Wait until @elastic/eui is at least version 13.7.0 to use display="block"
-        // display="block"
-        button={
+    <EuiPopover
+      id="lnsFieldListPanel__field"
+      // Wait until @elastic/eui is at least version 13.7.0 to use display="block"
+      display="block"
```

Comment:
We really do need display="block" here, so I'm hoping to get #45052 merged into master first